src/waltz/quic/fd_quic_private.h

```diff
@@ -111,6 +111,9 @@ struct __attribute__((aligned(16UL))) fd_quic_state_private {
 
   /* last arp/routing tables update */
   ulong ip_table_upd;
+
+  /* Scratch space for packet protection */
+  uchar                   crypt_scratch[FD_QUIC_MTU];
```

Comment:
I sized the buffer to allow for block ciphers... I suppose that isn't necessary here?